assets/css/edd-admin.css

```diff
@@ -2652,8 +2652,8 @@ body.download_page_edd-reports {
 #edd-reports-tiles-wrap {
 	display: flex;
 	flex-wrap: wrap;
-	margin-left: 10px;
-	margin-right: 10px;
+	margin-left: -13px;
+	margin-right: -10px;
```

Comment:
My first thought was to do negative margins too, but looking again it might be better to override the padding applied to `.edd-sections-wrap .section-wrap .section-content > *`

Perhaps we just override the padding to be `10px` which will be combined with the `10px` margin on the actual tiles. 

This also makes it easier because the padding override will only need to be applied on the breakpoint where the tiles go in to multiple columns (https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/assets/css/edd-admin.css#L2725)

Reply:
Since the styling here is by ID, overriding the padding is pretty straightforward. I would also see about adding `justify-content: center;` because the columns don't completely fill the space even with the calculated margins, so technically there is a bigger gap on the right than the left.

Reply:
I don't think the tiles are quite out to the edge of the other sections.  Personally I like how it looks, but I am happy to tweak as needed.